web/src/components/Feedback.tsx

```diff
@@ -88,7 +153,13 @@ const Feedback = (props: FeedbackProps): ReactElement => {
       {sendingStatus === SendingState.ERROR && (
         <ErrorSendingStatus role='alert'>{t('failedSendingFeedback')}</ErrorSendingStatus>
       )}
-      <TextButton disabled={!isPositiveFeedback && !comment} onClick={onSubmit} text={t('send')} />
+      {sendFeedbackDisabled && (
+        <NoteContainer>
+          <img src={NoteIcon} alt='' />
+          <NoteText>{t('note')}</NoteText>
+        </NoteContainer>
+      )}
```

Comment:
Could we just hide the note instead of unrendering it completely? That way, the height of the modal/position of the send button wouldn't jump.

Reply:
just set the opacity to 0 or 1